README.md

```diff
@@ -66,18 +66,22 @@ The `samlconf` script may take the following parameters:
            not provided, the default values will use DDF's parameters.
     
     OPTIONS
-           -i | --implementation path
+           -i path
```

Comment:
❓ Why are we getting rid of the `--` options?

Reply:
It doesn't work